src/site/content/en/fast/debugging-layout-shifts/index.md

```diff
@@ -0,0 +1,457 @@
+  height="354"
+ %}
+
+{% Aside %}
+The duration of a layout shift has no impact on its layout shift score.
+{% endAside %}
+
+For more information on using the Performance Panel, refer to [Performance
+Analysis
+Reference](https://developers.google.com/web/tools/chrome-devtools/evaluate-performance/reference).
+
+
+#### Highlight layout shift regions
+
+Highlighting layout shift regions can be a helpful technique for getting a
+quick, at-a-glance feel for the location and timing of the layout shifts
+occurring on a page.
+
+To enable Layout Shift Regions in DevTools, go to **Settings > More Tools >
+Rendering > Layout Shift Regions** then refresh the page that you wish to debug.
+Areas of layout shift will be briefly highlighted in purple.
+
+
+## Thought Process
+
+The steps listed below can be used to identify the cause of layout shifts
+regardless of when or how the layout shift occurs. These steps can be
+supplemented with running Lighthouse—however, keep in mind that Lighthouse can
+only identify layout shifts that occurred during the initial page load. In
+addition, Lighthouse also can only provide suggestions for some causes of layout
+shifts—for example, image elements do not have explicit width and height.
+
+
+### Identifying the cause of a layout shift
+
+Layout shifts can be caused by the following events:
+
+
+
+*   Changes to the position of a DOM element
+*   Changes to the dimensions of a DOM element
+*   Insertion or removal of a DOM element
+*   Animations that trigger layout
+
+In particular, the DOM element immediately preceding the shifted element is the
+element most likely to be involved in "causing" layout shift. Thus, when
+investigating why a layout shift occurred consider:
+
+
+
+*   Did the position or dimensions of the preceding element change?
+*   Was a DOM element inserted or removed before the shifted element?
+*   Was the position of the shifted element explicitly changed?
+
+If the preceding element did not cause the layout shift, continue your search by
+considering other preceding and nearby elements.
+
+In addition, the direction and distance of a layout shift can provide hints
+about root cause. For example, a large downward shift often indicates the
+insertion of a DOM element, whereas a 1px or 2px layout shift often indicates
+the application of conflicting CSS styles or the loading and application of a
+web font.
+
+<figure class="w-figure">
+  {% Img
+    src="image/j2RDdG43oidUy6AL6LovThjeX9c2/g0892nhvz3SnSaasaO1b.png",
+    alt="Diagram showing a layout shift caused by a font swap",
+    width="800",
+    height="452"
+  %}
+  <figcaption class="w-figcaption">
+    In this example, font swapping caused page elements to shift upwards by five pixels.
+  </figcaption>
+</figure>
+
+
+
+These are some of the specific behaviors that most frequently cause layout shift
+events:
+
+#### Changes to the position of an element (that aren't due to the movement of another element)…
+
+Are often the result of:
```

Comment:
This formatting might sound weird on screen readers as they often announce the heading levels. I'd like to check this before merging (probably tmrw)